### ISSUE TITLE
Determine path parameters needed for successful scheduling either locally or on server

### ISSUE THREAD

**Issue by CatChenal** [GunnerLab/MCCE_Benchmarking#38]
This f-string used to build the crontab, works on WSL, but not on the server:
```
SINGLE_CRONTAB = """PATH={}:{}:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:
* * * * * {}/conda activate {}; bench_launchjob -bench_dir {} -job_name {} -n_batch {} -sentinel_file {}"""
```
Fails with "bench_launchjob not found" or "python no in usr/bin/env".

 * Test 1: 
   1. Add a slot for python to be found in either the base env or the user env (i.e. in 'bin/' or 'condabin/').
   2. Use 'source <conda path>/activate <env>' instead of <conda path>/conda activate <env>'

 * Implementation:
```
# in case user is using 'base' env:
PATH_1 = "PATH={}:{}:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:\n"

# with dedicated env:
PATH_2 = "PATH={}:{}:{}:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:\n"

# schedule:
SCHED = "* * * * * source {}/activate {}; bench_launchjob -bench_dir {} -job_name {} -n_batch {} -sentinel_file {}\n"
```

**Comment by CatChenal** [comment]
To fix "bench_launchjob: not found" error from cron, need to add full path to it:
```
<conda env path>/bin/bench_launchjob
```

**Comment by CatChenal** [comment]
Issue won't be resolved until [Stable-MCCE #283](https://github.com/GunnerLab/Stable-MCCE/issues/283) is.

**Comment by CatChenal** [comment]
Crontab creation finalized by commit https://github.com/GunnerLab/MCCE_Benchmarking/commit/69c54f5022a584275b84e41c2a789051db0b90c4.
